usaspending_api/etl/helpers.py

```diff
@@ -204,20 +204,34 @@ def update_model_description_fields():
         # default updates are stored
         model_filtered_update_case_map = [(Q(), {})]
 
+        desc_fields = [field for field in model_fields if field.split('_')[-1] == "description"[:len(field.split('_')[-1])]]
+        non_desc_fields = [field for field in model_fields if field not in desc_fields]
+        desc_fields_mapping = {}
+        for desc_field in desc_fields:
+            actual_field_short = "_".join(desc_field.split('_')[:-1])
+            actual_field = None
+            for field in non_desc_fields:
+                if actual_field_short == field:
+                    actual_field = field
+                elif actual_field_short == field[:len(actual_field_short)]:
+                    actual_field = field
+            desc_fields_mapping[desc_field] = actual_field
+
         # Loop through each of the models fields to construct a case for each
         # applicable field
         for field in model_fields:
             # We're looking for field names ending in _description
             split_name = field.split("_")
 
             # If the last element in our split name isn't description, skip it
-            if len(split_name) == 1 or split_name[-1] != "description":
+            if len(split_name) == 1 or split_name[-1] != "description"[:len(split_name[-1])]:
```

Comment:
It took me a bit to figure out what this was doing.  Maybe update the comment to say "If the last element in our split name isn't description or an abbreviation for description"?

Also, is there any risk that we'll accidentally pick up something that isn't meant to be a description?

```
field = 'german_language_field_de'

split_name = field.split("_")

split_name[-1] == "description"[:len(split_name[-1])]```